projects/kubernetes/kubernetes/1-20/patches/0009-EKS-PATCH-aws_credentials-update-ecr-url-validation-.patch

```diff
@@ -0,0 +1,46 @@
+From 80a1017709a8bf689d4d75b8df1970198d35ccb8 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?chrisDallas=20=E2=98=95=20=F0=9F=8C=A7=EF=B8=8F=20?=
+ =?UTF-8?q?=E2=98=82?= <cdalla@amazon.com>
+Date: Thu, 8 Oct 2020 11:58:58 -0700
+Subject: --EKS-PATCH-- (aws_credentials): update ecr url validation
+ regex
+
+Signed-off-by: Jackson West <jgw@amazon.com>
```

Comment:
Why is Jackson the sign-off for this one?

Reply:
```suggestion
Signed-off-by: Jyoti Mahapatra <jyotima@amazon.com>
```